core/src/main/java/io/substrait/expression/Expression.java

```diff
@@ -940,6 +940,7 @@ public static AggregationInvocation fromProto(AggregateFunction.AggregationInvoc
   }
 
   enum AggregationPhase {
+    UNSPECIFIED(io.substrait.proto.AggregationPhase.AGGREGATION_PHASE_UNSPECIFIED),
```

Comment:
There are probably a lot of switches that need to be updated appropriate.  A search for INITIAL_TO_RESULT found many potential instances across Java and Scala.

Reply:
The enumerations here are mapping from the protobuf enumerations; in this case here one of the valid values in the protobuf was missing from the Java enumerations; 

If a plan contained that it would fail immediately; which in the case of a plan coming from duckdb it did.

Reply:
> There are probably a lot of switches that need to be updated as appropriate. 

There's actually a Java warning enabled for exhaustive switch statements, which should trigger CI failures. I didn't see any pattern matching blocks in Java which would be affected.

I did find one in Scala running `./gradlew check` locally. I filed https://github.com/substrait-io/substrait-java/issues/321 for the fact that it didn't fail in CI.